source/technical/networks/networks_basic.rst

```diff
@@ -72,7 +72,15 @@ Firewall
 ^^^^^^^^
 
 Chameleon-supported Ubuntu and CentOS images are preconfigured with a firewall
-utility called ``firewalld`` enabled and the following rules set:
+utility called ``firewalld`` enabled.
+
+.. note::
+
+   The firewall is disabled on instances deployed on KVM since those instances
+   use security groups for the firewall. ``firewalld`` is enabled on all bare
+   metal instances in all other regions.
```

Comment:
I would note that this is the case for CC-images with a build-date after X
/ if you're using older releases of images, it won't automatically change

Reply:
done